guidelines.md

```diff
@@ -146,3 +147,11 @@ As an example, consider minimum and maximum length of the reservation:
 
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
+
+## Taxations
+
+Each enterprise is located in a specific [Tax environment](operations/configuration.md#tax-environment) that offers a list of applicable [Taxations](operations/configuration.md#taxation). The numeric value of the taxations are represented by [Tax rate codes](operations/configuration.md#tax-rate) that are accepted within the tax environment. 
```

Comment:
- enterprise could be a link
- The second sentence doesn't make sense to me - taxations don't have numeric values, tax rates have. Also we should always link the entities (tax rate) not their properties (TaxRate.Code).